src/utils/hederaUtils.js

```diff
@@ -89,6 +89,7 @@ module.exports = class HederaUtils {
   }
 
   static async generateECDSA(client, num, startup, logger) {
+    let accountNum = 1002;
```

Comment:
Could a flag be added named say `useFixedAccounts` which is defaulted to true and has the current behaviour of deterministic mapping of pvKey to accountId but if set to false creates all 30 accounts asynchronously as done in #165 ?

This would further speed up account creation from 5s to <1s.

Reply:
Open minded to this but i think it should be in a different PR as we'll want to document the effects of this config explicitly.
It's likely many will have assumptions of the deterministic behaviour so we'll want to make this clear in such a mode.